coding as a hobby.r

```diff
@@ -0,0 +1,18 @@
+library(readr)
+library(ggplot2)
+library(stringr)
+library(dplyr)
+library(plyr)
+
+
+
+
+survey_results_part<-read_csv("C:/Users/Danielleos/Desktop/COLLEGE/Year 2/Group Project/survey_results_part.csv")
```

Comment:
we need a workaround for this. We had one a while back, I can investigate. Not everybody will have the same directory structure.